website/client/src/components/tasks/taskModal.vue

```diff
@@ -1334,6 +1376,104 @@ export default {
     formattedDate (date) {
       return moment(date).format('MM/DD/YYYY');
     },
+    formattedDays (frequency, repeat, daysOfMonth, weeksOfMonth, startDate) {
+      let activeDays;
+      const dayStringArray = [];
+      switch (frequency) {
+        case 'weekly':
+          activeDays = keys(pickBy(repeat, value => value === true));
+          if (activeDays.length === 0) return ' on <strong>no days</strong>';
+          if (activeDays.length === 7) return ' on <strong>every day of the week</strong>';
+          dayStringArray.push(' on <strong>');
+          forEach(activeDays, (value, index) => {
```

Comment:
can we use the native `activeDays.forEach` instead of importing an extra lodash module?

Reply:
done!